ui/src/components/TaskDetailsCard/index.jsx

```diff
@@ -319,16 +320,20 @@ export default class TaskDetailsCard extends Component {
                     </ListItem>
                     <List disablePadding>
                       {dependentTasks.map(task => (
+                        // note that the task might not exist, if it has
+                        // expired
                         <Link key={task.taskId} to={`/tasks/${task.taskId}`}>
                           <ListItem
                             button
                             className={classes.listItemButton}
                             title="View Task">
-                            <StatusLabel state={task.status.state} />
+                            <StatusLabel
+                              state={task.status?.state || 'EXPIRED'}
```

Comment:
is this our first `.?`?

Reply:
No, it's like the 7th -- owlish used it in a few other places (which is how I learned about it)